### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
- Follow design 
- Use React components.
- Use React props.
- Use React state.
- Remove boilerplate

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
```
Comment (D4nielJ) [inline src/App.js:1]: - [x] Please delete this logo too from the src folder as it's no longer in use and it was part of the boilerplate that CRA creates.

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,8 @@
...
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```
Comment (D4nielJ) [inline src/App.js]: - [x] Please make sure that all of the components from the application are class-based components as required from Microverse. This includes the `App` component and the `CalculatorLayout` component you have created. 

I know this doesn't much sense right now, as components with no state should be functional components, but the objective here it's to practice and understand React on a deeper level.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Lucas-Masaba, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Lucas-Masaba) [reply src/App.js:1]: Thank you for that observation. I have deleted this svg

File: src/App.js (hunk 2 above)
Comment (Lucas-Masaba) [reply src/App.js]: Thank you again. I changed all the components to class-based components.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Lucas-Masaba, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**